_stories/molecules/RadioGroup/index.js

```diff
@@ -26,6 +26,7 @@ const component = () => (
         context={optionalSelect('Context', contextOptions, '')}
         name="my-radio-group"
         options={[{ label: 'Foo', value: 'foo' }, { label: 'Bar', value: 'bar' }]}
+        defaultValue="bar"
```

Comment:
can you make this changeable in storybook? `defaultValue={text('defaultValue', 'bar'}`
and you need to `import { text } from '@storybook/addon-knobs';`

Reply:
Indeed, I tried at first and it wasn't working